posts/2023-07-24_rounding/rounding.qmd

```diff
@@ -0,0 +1,142 @@
+---
+title: "Rounding"
+author:
+  - name: Kangjie Zhang
+description: "base R `round()` and round half up in R"
+date: "2023-07-24"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [rounding]
+# feel free to change the image
+image: "rounding.png"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2023-07-24_rounding"
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+## Rounding methods
+
+Both SAS and base R have the function `round()`, which rounds the input to the specified number of decimal places.
+However, they use different approaches when rounding off a 5:
+
+-   SAS `round()` rounds half up (the common method of rounding)
```

Comment:
how about changing to the following? (to avoid referring to "round up" and "round half up" as the same method) -
SAS `round()` [rounds half up](https://en.wikipedia.org/wiki/Rounding#Rounding_half_up). This is the most common method of rounding.